src/index.js

```diff
@@ -1,7 +1,295 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import L from 'leaflet';
+import borderData from './border.js';
+import countyData from './counties.js'
+import leafletPip from "@mapbox/leaflet-pip";
+
+class Page extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      gameStarted: false,
+      infoLoaded: false,
+      randomLocation: null,
+      upGiven: false,
+      countyGuess: null,
+      score: 0
+    }
+    this.handleStart = this.handleStart.bind(this);
+    this.handleGiveUp = this.handleGiveUp.bind(this);
+    this.handleCountySelect = this.handleCountySelect.bind(this);
+  }
+
+  handleStart () {
+    this.setState({gameStarted: true, upGiven: false, randomLocation: findVermontLocation(), infoLoaded: false, countyGuess: null})
+  }
+
+  componentDidUpdate ({ score }) {
+    if (this.state.infoLoaded===false) {
+      console.log(this.state.randomLocation)
+      fetch(`http://nominatim.openstreetmap.org/reverse.php?format=json&lat=${this.state.randomLocation.lat}&lon=${this.state.randomLocation.lng}`)
+        .then(response => response.json())
+        .then(object => findLocationInfo(object, this.state.randomLocation))
+        .then(this.setState({infoLoaded: true}))
+    }
+  }
+
+  handleGiveUp () {
+    this.setState({upGiven: true, gameStarted: false})
+  }
+
+  handleCountySelect () {
+    let countySelect = document.getElementById("county-select");
+    let guessWithCounty = countySelect.value + " County";
+    this.setState({countyGuess: countySelect.value, gameStarted: false})
+    if (guessWithCounty===this.state.randomLocation.county) {
+      let oldScore = this.state.score;
+      this.setState({score: oldScore + 10})
+    }
+  }
+
+  render() {
+    return (
+      <div id="site">
+        <h1>Geo Vermonter!</h1>
+        <div id="page">
+          <div>
+            <Map infoLoaded={this.state.infoLoaded} randomLocation={this.state.randomLocation}/>
+            <div id="page-bottom">
+              <div id="buttons-info">
+                <Buttons gameStarted={this.state.gameStarted} handleStart={this.handleStart} upGiven={this.state.upGiven} handleGiveUp={this.handleGiveUp} handleCountySelect={this.handleCountySelect}/>
+                <Info upGiven={this.state.upGiven} randomLocation={this.state.randomLocation} score={this.state.score} countyGuess={this.state.countyGuess}/>
+              </div>
+              <Controls />
+            </div>
+          </div>
+          <div>
+            <img src="./map.jpg" alt="Map of Vermont's counties"></img>
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+class Map extends React.Component {
```

Comment:
To improve your code organization, consider separating each class component into their own JavaScript files. Then, import these files from your index.js file.